viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -422,19 +407,6 @@ export class Cognite3DViewer {
    * ```
    */
   async addPointCloudModel(options: AddModelOptions): Promise<CognitePointCloudModel> {
-    trackLoadModel(
```

Comment:
Why don’t you want to track models loading anymore? Just wondering. Useless?

Reply:
No - we're tracking them also in `RevealManager` so this just caused duplicated events for the same models.